src/window.rs

```diff
@@ -168,7 +156,7 @@ pub struct WindowAttributes {
     pub fullscreen: Option<Fullscreen>,
     // Platform-specific configuration.
     #[allow(dead_code)]
-    pub(crate) platform_specific: PlatformSpecificWindowBuilderAttributes,
+    pub(crate) platform_specific: PlatformSpecificWindowAttributesAttributes,
```

Comment:
Nit: This has two `Attributes` in it.

```suggestion
    pub(crate) platform_specific: PlatformSpecificWindowAttributes,
```

Reply:
What happened here?

Reply:
I think maybe a rebase made GitHub close my comments automatically, and then I just assumed they were fixed?

Reply:
the branch is out of dated, so I resolved them but not pushed yet.